confchange/confchange.go

```diff
@@ -121,7 +121,7 @@ func (c Changer) LeaveJoint() (tracker.Config, tracker.ProgressMap, error) {
 }
 
 // Simple carries out a series of configuration changes that (in aggregate)
-// mutates the incoming majority config Voters[0] by at most one. This method
+// mutates the incoming majority config Voters.Incomming by at most one. This method
```

Comment:
Typos imply that you are doing a manual replacement, I'm worried that doing this manually will lead to errors that will be missed.

Reply:
I think it's ok since this replacement is very small，and all tests passed.